lib/task.js

```diff
@@ -183,13 +183,14 @@ export default class Task {
   */
   async dockerConfig(linkInfo) {
     let config = this.task.payload;
-    // Allow task specific environment vars to overwrite those provided by hooks
-    let env = _.defaults({}, config.env || {}, linkInfo.env || {});
 
     await this.runtime.privateKey.decryptEnvVariables(
       this.task.payload, this.status.taskId
```

Comment:
yea, I can change that.  What I actually want to do (but won't in this PR) is to pass it the payload, but it returns the envs back rather than doing magical things and modifying task.payload that is then used later.